### ISSUE TITLE
Mark Cree words in English definitions

### ISSUE THREAD

**Issue by aarppe** [UAlbertaALTLab/cree-intelligent-dictionary#706]
The English definitions have occasional Cree words, which should be explicitly marked and presented differently.

Example: _atimâ-pîsim_, cf.

<img width="772" alt="image" src="https://user-images.githubusercontent.com/15003407/111547941-8771be80-873f-11eb-8a43-9292ccecc2b9.png">

**Comment by aarppe** [comment]
@dwhieb @DBDacanay I believe you've already been doing this, but is it only in MD or also in CW?

**Comment by dwhieb** [comment]
@aarppe For MD, @DBDacanay extracted _examples_ and _notes_. There still may be a rare case where a Cree word is actually part of the core definition, but generally speaking there won't be any Cree words in the MD definitions now.

Same goes for CW. I'm extracting notes and examples, but not Cree words that are part of the core definition. Those will simply require manual marking and review. I'm not sure how many cases there are yet.

**Comment by andrewdotn** [comment]
I think we should wait until there is frontend dev time to design the UI aspects in tandem with the backend data.

As a data structure, I can suggest attributed text: an array containing a list of:
  - strings, or
  - objects with a `text` field, `type` field, and type-specific attributes

But if going that route, we’d need to define what types they are and how they’d be displayed.

```
  "rich_definition": [
    {
      "text": "Alkaid",
      "type": "link",
      "href": "https://en.wikipedia.org/wiki/Eta_Ursae_Majoris"
    },
    ", star at the end of the handle of ",
    { "text": "ocêk-atâhk", "type": "xref", "slug": "ocêk-atâhk" }
  ]
```

However this might not necessarily be appropriate.

**Comment by aarppe** [comment]
@dwhieb This should be addressed in #982?

**Comment by dwhieb** [comment]
@aarppe No, #982 handles cross-reference _notes_ in definitions (between [square brackets]), but not Cree words that are part of the core definitions. Cree words inside definitions proper will either need to be manually marked by the by lexicographer, or will need to be manually converted to a cross-reference note in square brackets (again by the lexicographer).

If we want to do this, the first step is to settle on a convention for marking up source language words within definitions. One possibility is to follow Markdown conventions and use a link, with value of the link pointing to a slug for another entry, like so:

```md
Alkaid, star at the end of the handle of [ocêk-atâhk](https://itwewina.altlab.app/word/ocêk-atâhk/), the Big Dipper
```

Then, when converted to the DLx database format, that could be stored in a `richDefinition` field as HTML:

```json
  "richDefinition": "Alkaid, star at the end of the handle of <a href=https://itwewina.altlab.app/word/ocêk-atâhk/>ocêk-atâhk</a>, the Big Dipper"
```

That in turn would be displayed in morphodict as a link:

> Alkaid, star at the end of the handle of [ocêk-atâhk](https://itwewina.altlab.app/word/ocêk-atâhk/), the Big Dipper
